src/middlewares/validateEntry.js

```diff
@@ -0,0 +1,23 @@
+const expenseService = require('./../services/expense.service');
+const userService = require('./../services/user.service');
+
+const validateEntry = (entryType) => {
```

Comment:
Same, `validateEntryExists`
Just so it is perfectly clear what this function should do and when to be used.
`validateEntry`, can sound like some function that would be used also for creating an entry. However, this one validates only if an entry already exists, thus `validateEntryExists`